gradle/libs.versions.toml

```diff
@@ -1,12 +1,12 @@
 [versions]
 cacheApiVersion = "1.1.1"
-springDataVersion="3.1.8"
-springFrameworkVersion = "6.0.16"
+springDataVersion="3.1.10"
+springFrameworkVersion = "6.0.18"
 springShiroVersion = "1.12.0"
 springShellVersion = "1.2.0.RELEASE"
-springBootVersion = "3.1.8"
+springBootVersion = "3.1.9"
```

Comment:
Let's hold off on merging this one... with Spring Boot 3.1.10 being released a little later this week